scripts/generate-renode-scripts.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python3
+
+import csv
+import zlib
+import argparse
+
+def generate_ethmac(data):
+    base = get_entry_value(data, 'csr_base', 'ethmac')
+    buffer_base = get_entry_value(data, 'memory_region', 'ethmac')
+    buffer_size = get_entry_value(data, 'memory_region', 'ethmac', 1)
+    interrupt = get_entry_value(data, 'constant', 'ethmac_interrupt')
+
+    result = [
+        'ethmac: Network.LiteX_Ethernet @ {',
+        '       sysbus <{}, +0x100>;'.format(base),
+        '       sysbus new Bus.BusMultiRegistration {{ address: {}; size: {}; region: "buffer" }}'.format(buffer_base, buffer_size),
+        '    }'
+    ]
+
+    if interrupt is not None:
+        result.append('    -> cpu@{}'.format(interrupt))
+    return result
+
+def generate_peripheral(data, name, model, properties=None):
```

Comment:
What if the peripheral doesn't exist?

Reply:
My idea was to return an empty array that would concatenate nicely, but now the generating code is totally reworked, so this is no more valid.